cmd/containerd-shim-runhcs-v1/task_hcs.go

```diff
@@ -152,18 +163,10 @@ func createContainer(
 			Spec:             s,
 			HostingSystem:    parent,
 			NetworkNamespace: netNS,
+			LCOWLayers:       lcowLayers,
+			WCOWLayers:       wcowLayers,
```

Comment:
I currently haven't done this because it didn't seem to provide much benefit, plus we will have to add another empty `Layers` interface and that would make the code more confusing. But let me know if you think otherwise.